services/scripts.py

```diff
@@ -1,61 +1,66 @@
 # Music recommendation application
 import csv
 
-from cfg import TableEnum
+from cfg import TableIndexes
 
 
 class Song:
     def __init__(self, row: list[str]) -> None:
-        self.id = int(row[TableEnum.ID.value])
-        self.name = row[TableEnum.Track.value]
-        self.album = row[TableEnum.Album.value]
-        self.artist = row[TableEnum.Artist.value]
+        self.id = row[TableIndexes.ID.value]
+        self.name = row[TableIndexes.TRACK.value]
+        self.album = row[TableIndexes.ALBUM.value]
+        self.artist = row[TableIndexes.ARTIST.value]
 
 
 class DataBase:
-    def __init__(self, filename):
+    def __init__(self, filename: str) -> None:
         self.db = {}
 
         with open(filename, encoding='utf-8-sig') as file:
             _ = file.readline()
             reader = csv.reader(file)
 
             for row in reader:
-                self.add_song(Song(row))
+                self.add_song(self.__parse_row(row))
 
-    def add_song(self, song: Song) -> None:
-        self.db[song.id] = song
+    @staticmethod
+    def __parse_row(row: list[str]) -> Song:
```

Comment:
This is privet method, used sing _ sign, not __. More here: [https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name](click)

Reply:
Yeh, I know, but I want that this method not to be used in child classes.

Reply:
> This is privet method, used sing _ sign, not __. More here: [https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name](click)


i don’t thing it matters, in this case `__parse_row` should only be used in the `DataBase` class, so, `__…` can be here

Reply:
If it doesn't matter, then private or superprivate don't matter.

> in this case `__parse_row` should only be used in the `DataBase` class

So... In this case superprivate is more suitable...